types/asana/index.d.ts

```diff
@@ -382,7 +360,6 @@ declare namespace asana {
              *     be either the object returned from an access token request (which
              *     contains the token and some other metadata) or just the `access_token`
              *     field.
-             * @constructor
```

Comment:
`@constructor` is unnecessary when decorating a `new` signature. It's implied by it being `new`. See also `@class`.